install/jaspic/bin/ts.jte

```diff
@@ -16,10 +16,10 @@
  
 #########################################################################
 ##
-##   JavaTest Environment file for Java EE Compatibility Test Suite
+##   JavaTest Environment file for Jakarta EE Compatibility Test Suite
```

Comment:
I think that ```Jakarta EE Compatibility Test Suite``` should instead be ```Jakarta EE Platform TCK```.

Looks like there are around 114 references to ```Compatibility Test Suite``` in the Platform TCK, I do regret that we didn't start the TCK documentation changes earlier.

Reply:
Sounds good. I'll make the change. Indeed, throughout the TCK there are tons of references to terms such as these. At least it'll be a few less now ;)